spec/models/metadatum_spec.rb

```diff
@@ -6,4 +6,7 @@
   # Nothing to test. Its all incredibly simple, test coverage is provided by:
   # - The FactoryBot lint tests
   # - The event specs, which provide a degree of integration testing
+  it 'can be created' do
+    expect(described_class.new).to be_a(described_class)
+  end
```

Comment:
Rubocop didn't like the empty spec file. There really isn't anything going on here, but lets just test it doesn't blow up catestrophically.